apps/analogquadclk/app.js

```diff
@@ -86,6 +86,21 @@ function draw() {
   lastModified = getHandBounds();
   //g.drawRect(lastModified); // debug
 }
+// Show launcher when middle button pressed
```

Comment:
Was there a reason for moving this upwards?

It's just that if (it's unlikely I know) `setUI` gets called again after this, clockInfoMenuA/etc aren't defined at this point.